helm-charts/charts/kube-starrocks/charts/operator/templates/clusterrole.yaml

```diff
@@ -54,18 +54,21 @@ rules:
   - starrocks.com
   resources:
   - starrocksclusters
+  - starrockswarehouses
   verbs:
   - '*'
 - apiGroups:
   - starrocks.com
   resources:
   - starrocksclusters/finalizers
+  - starrockswarehouses/finalizers
```

Comment:
is it still necessary?

Reply:
I'm not sure about the purpose of finalizers, so I have kept it consistent. I will confirm later whether this permission is needed.